openedx/core/djangoapps/call_stack_manager/tests/tests.py

```diff
@@ -0,0 +1,162 @@
+"""
+Test cases for Call Stack Manager
+"""
+from .models import ModelMixin, ModelNothing, ModelMixinCSM, ModelAnotherCSM
+from django.test import TestCase
+from testfixtures import LogCapture
```

Comment:
If you're going to use this, you have to make sure it's installed with the platform. However, rather than adding a new dependency, I would suggest using `@patch('openedx.core.djangoapps.call_stack_manager.core.log')` instead, and then counting calls to particular methods on the mocked logger.